code/src/components/NotFound.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { useHistory } from 'react-router-dom';
+import styled from 'styled-components';
+
+// PAGE NOT FOUND COMPONENT (START PAGE)
+const NotFound = () => {
+  const history = useHistory();
+
+  const onHomePageRedirect = () => {
+    history.push('/');
```

Comment:
Here you are creating the same function as you did in Details.js component. Maybe it's not necessary to both have the button-functions and buttons on two places? It's a bit hard to read the code.

Reply:
Yes, totally true, thanks for pointing that out!